Shared/Samples/Add feature layer with time offset/AddFeatureLayerWithTimeOffsetView.swift

```diff
@@ -63,58 +63,54 @@ struct AddFeatureLayerWithTimeOffsetView: View {
+                    .clipShape(.rect(cornerRadius: 10))
+                    .shadow(radius: 50)
+                    .padding()
                 }
-                .fixedSize()
-                .padding()
-                .background(.thinMaterial)
-                .clipShape(.rect(cornerRadius: 10))
-                .shadow(radius: 50)
-                .padding()
-            }
-            .toolbar {
-                ToolbarItem(placement: .bottomBar) {
-                    VStack {
-                        Slider(value: $sliderValue, in: 0...sliderMaxValue, step: 1)
-                            .padding(.horizontal)
-                            .onChange(of: sliderValue) {
-                                // Calculates a new time extent when the slider value changes.
-                                guard let newStartDate = Calendar.current.date(
-                                        byAdding: .day,
-                                        value: Int(sliderValue),
-                                        to: .august4th2000
-                                      ),
-                                      let newEndDate = Calendar.current.date(
-                                        byAdding: .day,
-                                        value: 10,
-                                        to: newStartDate
-                                      ) else {
-                                    return
-                                }
-                                timeExtent = TimeExtent(startDate: newStartDate, endDate: newEndDate)
-                            }
-                        Text(
-                            """
-                            \(timeExtent?.startDate?.formatted(dateFormat) ?? "") - \
-                            \(timeExtent?.endDate?.formatted(dateFormat) ?? "")
-                            """
-                        )
+            Slider(value: $sliderValue, in: 0...sliderMaxValue, step: 1)
+                .padding(.horizontal)
+                .onChange(of: sliderValue) {
+                    // Calculates a new time extent when the slider value changes.
+                    guard let newStartDate = Calendar.current.date(
+                        byAdding: .day,
+                        value: Int(sliderValue),
+                        to: .august4th2000
+                    ),
+                          let newEndDate = Calendar.current.date(
+                            byAdding: .day,
+                            value: 10,
+                            to: newStartDate
+                          ) else {
+                        return
                     }
+                    timeExtent = TimeExtent(startDate: newStartDate, endDate: newEndDate)
                 }
-            }
+            Text(
+                """
+                \(timeExtent?.startDate?.formatted(dateFormat) ?? "") - \
+                \(timeExtent?.endDate?.formatted(dateFormat) ?? "")
```

Comment:
`timeExtent` is optional, but it looks like it doesn't need to be. Also, as far as I can tell, there will always be a start date and an end date, so those could just be force unwrapped. Lastly, formatting could (and should) be done using format styles:

```swift
\(timeExtent.startDate!, format: dateFormat) - \
\(timeExtent.endDate!, format: dateFormat)
```